tests/filesystem/clean_path_test.php

```diff
@@ -1,54 +0,0 @@
-<?php
```

Comment:
I think you should keep both the old and new tests (even if they look duplicated)